### ISSUE TITLE
Basic web clipper functionality

### ISSUE THREAD

**Issue by VictorVow** [timediv/QuickDynalist#16]
The same text you would be able to clip from a webpage if you put it in Reader Mode (text-only), it would be very helpful to clip that into Dynalist. 

So it would look like:
[Item title] blah blah webpage title
[Item notes area] URL of the webpage
- child item 1 with first line of webpage content 
- child item 2 with second line of webpage content

**Comment by louiskirsch** [comment]
You can already select text, then select the three dots, then chose 'Add to Inbox'. Are you aware of that feature?
It does not save the url though.

**Comment by VictorVow** [comment]
Yes I'm aware of it, this is just a quicker way of doing so since it saves you typing out the webpage title manually and copying the URL also in your example to get my example's formatting. Basically "Add to Inbox - webpage" as a second option.

**Comment by louiskirsch** [comment]
Oh I might have misunderstood.
You want to copy in the entire webpage text given a URL?

**Comment by VictorVow** [comment]
Yes
